docs/tutorials/02-channel-upgrades/03-run-chains.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Run 2 Cosmos SDK Blockchain Locally
+sidebar_label: Run 2 Cosmos SDK Blockchain Locally
+sidebar_position: 3
+slug: /channel-upgrades/run-chains
+---
+
+# Run 2 Cosmos SDK blockchains locally
+
+The gm tool uses a [configuration file](https://github.com/informalsystems/gm/blob/master/gm.toml). This tutorial uses the following configuration file for gm:
+
+```yaml title="gm.toml"
+[global]
+add_to_hermes=true
+home_dir="~/testing/gm"
+
+[global.hermes]
+binary="~/testing/hermes/hermes"
+config="~/testing/hermes/config.toml"
+
+[chain1]
+  gaiad_binary="~/testing/bin/chain1/simd"  
+  ports_start_at=27000
+
+[chain2]
+  gaiad_binary="~/testing/bin/chain2/simd" 
+  ports_start_at=27010
+```
+
+The configuration file needs to be placed in `$HOME/.gm`. This configuration file sets up 2 blockchains (`chain1` and `chain2`), each with 2 accounts (1 validator, 1 wallet). The ports where the CometBFT RPC interface for each chain is 27000 for `chain1` and 27010 for `chain2`.
+
+In order to shorten the voting period of governance proposal, we are going to change some of the `x/gov` module parameters in the `genesis.json` file, so that we can complete the upgrade faster. These are the changes needed in the `genesis.json` of `chain1`:
+
+```json title="genesis.json"
+"gov": {
+  "starting_proposal_id": "1",
+  "deposits": [],
+  "votes": [],
+  "proposals": [],
+  "deposit_params": null,
+  "voting_params": null,
+  "tally_params": null,
+  "params": {
+    "min_deposit": [
+      {
+        "denom": "stake",
+// minus-diff-line
+-       "amount": "10000000"
+// plus-diff-line
++       "amount": "100"
```

Comment:
since we've already suggested they install jq, I think we can actually provide some jq commands to make these modifications.

This would be just a small QOL change, will leave it to you if you want to include them!

Reply:
My jq-foo is not very strong, so not to spend too much time on it, I think I will leave it as is for now...